packages/core/src/config.ts

```diff
@@ -725,6 +724,11 @@ const composeTargetConfig = (
         tools: {
           rspack: {
             target: ['web'],
+            output: {
```

Comment:
What are the problems with inferring based on target and chunkformat?

Reply:
> need to manually assign {type}loading as it derive the value from target by default, which will lead to unimpelmented universal type.